### ISSUE TITLE
PSA: last lib32-zstd package update broke mesa compilation

### ISSUE THREAD

**Issue by Tk-Glitch** [Tk-Glitch/PKGBUILDS#490]
The libdir path is wrong, it points to `/usr/lib` instead of `/usr/lib32`.
Here's a fixed precompiled package: https://cdn.discordapp.com/attachments/517945839520448514/676086354752307280/lib32-zstd-1.4.4-1-x86_64.pkg.tar.zst
And here's the PKGBUILD if you want to build it yourself: https://cdn.discordapp.com/attachments/517945839520448514/676086463338905610/PKGBUILD

I have reported the issue upstream btw, so let's hope for a quick fix.
Here's the culprit: https://github.com/facebook/zstd/pull/1851

**Comment by gardotd426** [comment]
lol I was just about to run to the issues page and file an issue for this. But yep, using the `PKGBUILD` in the comment above fixes it! Also, you know how I am having issues with my 5600 XT in Arch where menu text in RE2 Remake and RE7 wasn't rendering? Well I installed Pop OS in addition to my Arch install, and after adding the new firmware packages and using the oibaf PPA mesa-git 20.1.0_devel, the menu text in BOTH RE2 Remake and RE7 renders perfectly fine! It's driving me crazy, that's why I was gonna try your mesa-git builds and see if that helped with anything. 

It's like, there's a bunch of stuff that only works on Pop OS even with the same wine/proton/dxvk version and same Mesa version, but then there's other stuff (like `vkd3d`) that only works on Arch and I can't get to work on Pop OS, smh. It sucks that I absolutely refuse to use Windows lol

**Comment by Tk-Glitch** [comment]
Did you try with updated firmware on Arch? The package was updated in the testing repo only 2 days ago. The core package "only" contains release firmware for the 5600XT right now.

**Comment by gardotd426** [comment]
Well I mean, I downloaded it from the developers way before it hit any repos, and what's more, I'm using the firmware from Arch in Pop right now too. So that's not it. Hell, Pop OS wouldn't even boot with the 5600 XT until I added the firmware from Arch, I filed an issue with Pop OS saying the 5600 XT wouldn't boot and I got the ol' "The 5600 XT is not yet supported on Pop OS, it should be in 20.04 tho lol." I had to do it myself, so I booted into arch, mounted the pop root partition, copied the firmware files that I got from git, booted into pop, added the oibaf PPA, installed mesa-dev 20.1.0-git and was up and running.

**Comment by Tk-Glitch** [comment]
Same kernel version? I just find it extremely weird that the 5700 series isn't affected on the same OS with the same driver when it's the exact same GPU. I'll continue to think something is frogged on your Arch install 🐸

**Comment by gardotd426** [comment]
Did I not mention that it does it on Manjaro too? Yeah Manjaro has the
exact same issues as Arch with RE2/RE7. That's why I don't think it's
something frogged, because Manjaro is definitely a lot less easy to mess
up. I installed Manjaro, installed all the stuffs, and that's it. I've
never messed around with any system files with Manjaro. And yet, same
problem. And yeah it's all kernels on Manjaro/Arch (some of which are the
literal exact same kernel), ranging form vanilla Arch 5.5 and vanilla
Manjaro 5.5, and then all your custom ones I have installed as well. On
Pop, since y'know, it doesn't have those, I installed the liquorix 5.5
kernel and that's where RE2 and RE7 work with none of the menu text issues.
